client-side/src/App.jsx

```diff
@@ -1,18 +1,32 @@
 import { Provider } from 'react-redux';
-import { RouterProvider } from 'react-router-dom';
+import { HashRouter, Routes, Route } from 'react-router-dom';
 import Footer from './stories/footer/FooterComponent';
-import { router } from './router/router.jsx';
 import { store } from './redux/store.jsx';
+import Layout from './router/layout.jsx';
+import ProfileList from './components/profileComponent.jsx';
 import './App.scss';
 
+
 function App() {
   return (
-    <>
-      <RouterProvider router={router} />
-      <Provider store={store}>
+    <Provider store={store}>
+      <HashRouter>
+        <Routes>
+
+          <Route path="/" element={<Layout />}>
+            <Route index element={<h1>home</h1>} />
+            <Route path="home" element={<h1>home</h1>} />
```

Comment:
please save titles in constants file

Reply:
these titles are just for the experiment until there are real pages to redirect to
this is to avoid errors

Reply:
so please add TODO